File: conformance-tests/statements/for-loops/simple/nested-for-loops-modifying-temp_m5.lua
do
	local i = 0
	while
		i
		< 3 --[[ ROBLOX CHECK: operator '<' works only if either both arguments are strings or both are a number ]]
	do
		foo(i)
		if i == 3 then
			i += 1
		end
		do
			local j = 0
			while
				j
				< 5 --[[ ROBLOX CHECK: operator '<' works only if either both arguments are strings or both are a number ]]
			do
				if i == j then
					j *= 2
				end
				bar(i, j)
				j += 1
			end
		end
		i += 1
	end
end

File: conformance-tests/statements/for-loops/simple/nested-for-loops-modifying-temp_m5.js
for (let i = 0; i < 3; i++) {
  foo(i)
  if (i === 3) {
    i += 1;
  }
  for (let j = 0; j < 5; j++) {
    if (i === j) {
      j *= 2
    }
    bar(i, j)
  }
}
